frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCards/FeatureStrategyMenuCards.tsx

```diff
@@ -76,14 +126,18 @@ export const FeatureStrategyMenuCards = ({
                         <StyledTypography color='textSecondary'>
                             Release templates
                         </StyledTypography>
-                        {templates.map((template) => (
-                            <ListItem key={template.id}>
-                                <FeatureReleasePlanCard
-                                    template={template}
-                                    onClick={() => onAddReleasePlan(template)}
-                                />
-                            </ListItem>
-                        ))}
+                        <GridSection>
+                            {templates.map((template) => (
+                                <CardWrapper key={template.id}>
+                                    <FeatureReleasePlanCard
+                                        template={template}
+                                        onClick={() =>
```

Comment:
Is there any way we can make this a button or an appropriate input element instead? Butting click handlers on divs is real iffy in terms of accessibility and stuff. Are they reachable by tab now, for instance? What does the screen reader announce when you focus them, etc. 

I've been trying to think what the most sensible setup here is. I'm not entirely sure, but buttons do support [flow content](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#flow_content) as children, so we can probably just chuck the entire card into the button. That feels kinda weird too, but at least we get the right roles etc for free. 

Happy for you to bring other ideas too. And of course: we can do it later, no worries 😄

Reply:
Or ... I don't know anymore. I trust Kitty Giraudel's writing on accessibility and she's got a post on [accessible cards](https://kittygiraudel.com/2022/04/02/accessible-cards/) (with links or buttons) that I think would be appropriate here (but the strategy / release plan name in a button, use css to make the whole thing clickable. 

But I understand if you don't wanna go through that dance. Personally I think it sounds fun, but hey, I'm kinda strange 😅